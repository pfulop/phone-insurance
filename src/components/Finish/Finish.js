import React from 'react';
import { Link } from 'react-router-dom';
import gCss from '../../index.css';
import css from './Finish.css';

export default () => (
  <div className={gCss.card}>
    <h1> Thank you </h1>
    <div className={css.intro}>
      We are processing your claim. We will contact you shortly
    </div>
    <div className={css.ready}>
      <Link to="/start-claim" className={css.startClaim}>
        I have lost another one
      </Link>
    </div>
  </div>
);
