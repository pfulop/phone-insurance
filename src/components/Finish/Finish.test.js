import React from 'react';
import Finish from './Finish';
import { shallow } from 'enzyme';

describe('Home', () => {
  it('should be defined', () => {
    expect(Finish).toBeDefined();
  });
  it('should render correctly', () => {
    const tree = shallow(<Finish />);
    expect(tree).toMatchSnapshot();
  });
});
