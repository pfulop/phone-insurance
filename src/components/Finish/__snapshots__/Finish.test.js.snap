// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
           Thank you 
</h1>,
        <div
          className={undefined}
>
          We are processing your claim. We will contact you shortly
</div>,
        <div
          className={undefined}
>
          <Link
                    className={undefined}
                    replace={false}
                    to="/start-claim"
          >
                    I have lost another one
          </Link>
</div>,
      ],
      "className": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": " Thank you ",
        },
        "ref": null,
        "rendered": " Thank you ",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "We are processing your claim. We will contact you shortly",
          "className": undefined,
        },
        "ref": null,
        "rendered": "We are processing your claim. We will contact you shortly",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Link
            className={undefined}
            replace={false}
            to="/start-claim"
>
            I have lost another one
</Link>,
          "className": undefined,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "I have lost another one",
            "className": undefined,
            "replace": false,
            "to": "/start-claim",
          },
          "ref": null,
          "rendered": "I have lost another one",
          "type": [Function],
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
             Thank you 
</h1>,
          <div
            className={undefined}
>
            We are processing your claim. We will contact you shortly
</div>,
          <div
            className={undefined}
>
            <Link
                        className={undefined}
                        replace={false}
                        to="/start-claim"
            >
                        I have lost another one
            </Link>
</div>,
        ],
        "className": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": " Thank you ",
          },
          "ref": null,
          "rendered": " Thank you ",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "We are processing your claim. We will contact you shortly",
            "className": undefined,
          },
          "ref": null,
          "rendered": "We are processing your claim. We will contact you shortly",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Link
              className={undefined}
              replace={false}
              to="/start-claim"
>
              I have lost another one
</Link>,
            "className": undefined,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "I have lost another one",
              "className": undefined,
              "replace": false,
              "to": "/start-claim",
            },
            "ref": null,
            "rendered": "I have lost another one",
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
